source/bringauto/structures/ModuleLibrary.cpp

```diff
@@ -11,9 +13,14 @@ ModuleLibrary::~ModuleLibrary() {
 				  [](auto &pair) { pair.second->destroy_status_aggregator(); });
 }
 
-void ModuleLibrary::loadLibraries(const std::unordered_map<int, std::string> &libPaths) {
+void ModuleLibrary::loadLibraries(const std::unordered_map<int, std::string> &libPaths, const std::string &moduleBinaryPath) {
+	std::shared_ptr<modules::IModuleManagerLibraryHandler> handler;
 	for(auto const &[key, path]: libPaths) {
-		auto handler = std::make_shared<modules::ModuleManagerLibraryHandler>();
+		if (moduleBinaryPath.empty()) {
+			handler = std::make_shared<modules::ModuleManagerLibraryHandlerLocal>();
```

Comment:
Here is the case handled. Maybe do an interface for ModuleLibrary and create two specialisations?